traktapi.py

```diff
@@ -343,6 +347,13 @@ def getIdLookup(self, id, id_type):
                 result = [result]
             return result
 
+    def getTextQuery(self, query, type, year):
+        with Trakt.configuration.http(retry=True):
```

Comment:
Sorry, my bad. I added that in the getShowWithAllEpisodesList function but didn't copy it here. Fixed.